src/OpenLoco/src/Map/Track/TrackData.h

```diff
@@ -59,6 +59,20 @@ namespace OpenLoco::World::TrackData
     const TrackCoordinates& getUnkTrack(uint16_t trackAndDirection);
     const TrackCoordinates& getUnkRoad(uint16_t trackAndDirection);
 
+    struct MiscData
+    {
+        uint16_t costFactor;             // 0x004F870C
+        uint16_t flags;                  // 0x004F8764
```

Comment:
flags and compatibleFlags are the same type. Should probably make a enum class for it. Its also the same type as disableTrackCfg in the object.

Reply:
> flags and compatibleFlags are the same type. Should probably make a enum class for it. Its also the same type as disableTrackCfg in the object.

no they are not. Flags and compatible flags are similar types but crucially different!